embassy-stm32/src/crc/v1.rs

```diff
@@ -0,0 +1,55 @@
+use crate::pac::CRC as PAC_CRC;
+use crate::peripherals::CRC;
+use crate::rcc::sealed::RccPeripheral;
+
+pub struct Crc {
+    _peripheral: CRC,
+}
+
+impl Crc {
+    /// Instantiates the CRC32 peripheral and initializes it to default values.
+    pub fn new(peripheral: CRC) -> Self {
+        // Note: enable and reset come from RccPeripheral.
+        // enable CRC clock in RCC.
+        CRC::enable();
+        // Reset CRC to default values.
+        CRC::reset();
+        let mut instance = Self {
+            _peripheral: peripheral,
+        };
+        instance.init();
+        instance
+    }
+
+    /// Resets the CRC unit to default value (0xFFFF_FFFF)
+    pub fn init(&mut self) {
```

Comment:
This is called `reset` in v2/v3. We should try to keep the API consistent across versions. Could you name this `reset`?